docs/development.md

```diff
@@ -49,23 +49,21 @@ Go to `http://keycloak.local.freshworks.club:8080/auth` console and add users/as
 - Run `make kc-export-users` to save your changes
 `409 Conflict` errors are ok if the user already exists, only the role change will be applied
 
-### To run server unit tests:
+### To run the testing suite:
 
-- Make sure containers are running
-  - `make local-kc-run`
-- Shell into the server container by running
-  - `make local-server-workspace`
-  - `npm test`
-- or by running
-  - `make local-server-tests`
-
-### To run e2e tests:
-- Make sure containers are running
-  - `make local-kc-run`
+- Whether or not you're currently running the app, run the test setup. This will ensure your server is pointed to the `db_test` database, clearing and repopulating it.
+  - `make local-test-setup`
```

Comment:
2 things I noticed while testing: 
1) the makefile command is defined as: `local-testing-setup` 
2) `make kc-import-users` needs to be run before 'make local-testing-setup' or there appears to be an issue regarding the participant's table and the setup fails. 

It might also be helpful to mention in the comment to set up local keycloak before running any of these commands, as that might not be obvious to new devs.